src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/sealedclaim/SealedClaimFormGeneratorForSpec.java

```diff
@@ -85,17 +86,10 @@ public CaseDocument generate(CaseData caseData, String authorisation) {
         );
     }
 
-    public byte[] downloadDocument(CaseDocument caseDocument) {
+    public ResponseEntity<Resource> downloadDocumentById(String documentId) {
```

Comment:
should just return a `Resource`

Reply:
done.
I created DownloadResponse object